src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -148,6 +149,7 @@ internal override bool IsExpressionBodied
             _explicitInterfaceImplementations = explicitInterfaceImplementations;
             _name = name;
             _isAutoPropertyAccessor = false;
+            _isExpressionBodied = true;
```

Comment:
Revied it. Because I override `IsExpressionBodied` to return the member field and not the `_property`'s `IsExpressionBodied` (that change is necessary!). Therefore `_isExpressionBodied` must be `true` in the constructor that takes an `ArrowExpressionClauseSyntax` as an argument.